SetupDataPkg/Docs/ConfigurationFiles/ConfigurationFiles.md

```diff
@@ -69,16 +70,20 @@ to denote an array ID tag value. [GenCfgData.py](../../Tools/GenCfgData.py) will
 content to backend database and generate the same binary data blob.
 
 - MU version of ConfigEditor supports 3 more option for loading from and saving to SVD (Setup Variable Data) files:
-  - Save Full Config Data to SVD File (supported for YAML/XML/Merged YAML + XML):
-      Saving the entire defined YAML structure into encoded binary settings format. This format is useful when many tags
-      of settings need updating at once, as it will save transmission overhead when serial method is selected. But this
-      will *touch* all configurations defined.
-  - Save Config Changes to SVD File (Supported for YAML):
+  - Save Full Config Data to SVD File:
```

Comment:
I think this section can be made clearer, and the recommended methods can be specified here.

we have 3 classes of file operations:

Var List Binary
           Save Config Data to Var List Binary
           Load Config Data from Var List Binary

Var List Binaries <expand on usecase here>

SVD File
           Safe Full Config Data to SVD File
           Save Config Changes to SVD File
           Load Config from SVD File

SVD files are used in conjunction with the ConfigApp that is part of the SetupDataPkg. 
<expand on usecase>

Change File
           Safe Full Config Data to Change File
           Save Config Changes to Change File
           Load Config Changes from Change file
<expand on usecase here>

Reply:
Sure, I'll add and clarify further